src/server/JasmineGraphInstanceService.cpp

```diff
@@ -1925,6 +1931,16 @@ bool JasmineGraphInstanceService::isInstanceDuplicateCentralStoreExists(std::str
     return true;
 }
 
+void JasmineGraphInstanceService::loadStreamingStore(std::string graphId, std::string partitionId, std::map<std::string,JasmineGraphIncrementalLocalStore>& graphDBMapStreamingStores) {
+    instance_logger.log("###INSTANCE### Loading streaming Store : Started", "info");
```

Comment:
Can we also have graph ID and partition ID information attached to this message? That ill give us the details on which local store got loaded.

Reply:
Yes, I have included graph identifier value (graph ID + partition ID) in the log string in this commit https://github.com/miyurud/jasminegraph/pull/132/commits/d8d887f5453408266df16065d675be6691184b72

Reply:
Ok